frontend/src/metabase/core/components/Radio/Radio.styled.tsx

```diff
@@ -66,7 +66,7 @@ export const RadioContainer = styled.div<RadioContainerProps>`
   &:hover {
     color: ${props =>
       !props.checked && !props.showButtons
-        ? getContrastSchemeColor(props.colorScheme)
+        ? getHoverColor(props.variant, props.colorScheme)
```

Comment:
I find it a bit hard to justify letting `<RadioContainer/>` know how to render hover color for **bubble** styled radio, as there is already `<RadioContainerBubble/>` which extends `RadioContainer`. And `<RadioContainer/>` itself is also extended by other radio styles namely  **normal** and **underline**

What do you think about defaulting this to
```suggestion
        ? getSchemeColor((props.colorScheme)
```

and set `<RadioContainerBubble/>`'s hover color using `getContrastSchemeColor()` instead. That way, the logic to handle **bubble** styled radio won't be inherited by **normal** and **underlined** variants. e.g.
```ts
    color: ${props =>
      !props.checked && !props.showButtons
        ? getContrastSchemeColor(props.colorScheme)
        : ""};
```

Reply:
@WiNloSt Fixed, please take a look